src/test/java/org/assertj/core/api/date/DateAssert_with_string_based_date_representation_Test.java

```diff
@@ -84,6 +84,21 @@ public void date_assertion_should_support_date_with_utc_time_zone_string_represe
     assertThat(date).isEqualTo("2003-04-26T00:00:00+00:00");
   }
 
+  @Test
+  public void date_assertion_should_support_date_with_different_utc_time_zone_string_representation() throws ParseException {
+    String dateFormatPattern = "yyyy-MM-dd'T'HH:mm:ss";
+    SimpleDateFormat isoDateFormatUtc = new SimpleDateFormat(dateFormatPattern);
+    isoDateFormatUtc.setTimeZone(TimeZone.getTimeZone("UTC"));
+
+    SimpleDateFormat isoDateFormatMadrid = new SimpleDateFormat(dateFormatPattern);
+    isoDateFormatMadrid.setTimeZone(TimeZone.getTimeZone("Europe/Madrid"));
+
+    Date date = isoDateFormatUtc.parse("2003-04-26T00:00:00");
+    String madridDate = isoDateFormatMadrid.format(date);
+
+    assertThat(date).isEqualTo(madridDate);
```

Comment:
this fails in travis ci:
```
DateAssert_with_string_based_date_representation_Test.date_assertion_should_support_date_with_different_utc_time_zone_string_representation:99 
expected:<2003-04-26T0[2]:00:00.000> but was:<2003-04-26T0[0]:00:00.000>
```
could it be that the picked date format is not the one supporting TZ ?

Reply:
Exactly. I replaced with Europe/Berlin.

Reply:
I guess this is not a `madridDate` anymore ;-)

Reply:
can we rename `madrid` stuff to `berlin` stuff is what I really wanted to say.

Reply:
Yes, but it failed with Berlin TZ in travis too.

Reply:
Yes, I will rename it.